Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,332 @@
+		* [Known Ringcode Information](#known-ringcode-information) ([Alt.](#Known%20Ringcode%20Information))
+			* [Mastering Code](#mastering-code)  ([Alt.](#Mastering%20Code))
+			* [Mastering SID Code](#mastering-sid-code)  ([Alt.](#Mastering%20SID%20Code))
+			* [Toolstamp or Mastering Code](#toolstamp-or-mastering-code)  ([Alt.](#Toolstamp%20or%20Mastering%20Code))
+			* [Mould SID Code](#mould-sid-code)  ([Alt.](#Mould%20SID%20Code))
+* [Reference Material](#reference-material) ([Alt.](#Reference%20Material))
+	* [Media](#media)
+	* [Additional Resources](#additional-resources) ([Alt.](#Additional%20Resources))
+	* [Footnotes](#footnotes)
+
+*** 
+
+## Categories: 
+
+* Optical Copy Protection
+
+***
+
+## Protection Features:
+
+* Ring protection[^Check_Versions_Light]
+* Dummy files[^Check_Versions_Light]
+* Disc check[^Check_Versions_Professional]
+* SoftICE Debugger Protection[^Check_Versions_Professional]
+
+[^Check_Versions_Light]: Please see the entry for "copy-X light" in "Versions & Differences" section.
+[^Check_Versions_Professional]: Please see the entry for "copy-X professional" in "Versions & Differences" section.
+
+***
+
+## DRML Entry Version: 
+
+***
+
+# DRM Information:
+
+## Names: 
+
+* copy-X[^CopyX_Original_Website]
+* copy-X - the ultimate Copy-Killer.[^CopyX_Original_Website]
+* copy-X audio[^CopyX_Original_Website]
+* copy-X light[^CopyX_Original_Website]
+* copy-X professional[^CopyX_Original_Website]
+* copy-X trial maker[^CopyX_Original_Website]
+
+[^CopyX_Original_Website]: [The original optimal media link for copy-X.](https://web.archive.org/web/20011016234742/http://www.optimal-online.de:80/product/copy_x.htm)
+
+***
+
+## Developers:
+
+* optimal media production[^CopyX_Original_Website] (Until 2011)[^Discogs_Optimal_Page]
+* optimal media[^CopyX_Current_Website] (Current)
+
+[^Discogs_Optimal_Page]: [Discogs page for optimal media](https://www.discogs.com/label/313190-Optimal-Media-GmbH)
+
+[^CopyX_Current_Website]: [The current optimal media website link.](https://web.archive.org/web/20241003002305/https://www.optimal-media.com/)
+
+***
+
+## Overall Description:
+
+Ring protection copy protection scheme from german company optimal media[^CopyX_Original_Website]. As seen in Names, four different types seem to have been in use[^CopyX_Original_Website]. Note that it has not yet been confirmed whether light and professional correspond to the currently established distinct behaviors, but it seems to be the most likely conclusion.
+Had to be mastered at a specific optimal media production plant[^CopyX_2009_Document].
+
+[^CopyX_2009_Document]: [Optimal media document from 2009, seemingly for prospective customers.](https://web.archive.org/web/20220322025111/http://press9.de/fileadmin/user_upload/PDF/copy_protection_july_09.pdf)
+
+***
+
+## Timeline: 
+
+#### 2000s
+
+* 2001?: Seems to come into existence about here[^CopyX_Original_Website].
```

Comment:
First known reference to copy-X is technically here: https://web.archive.org/web/20000302195240/http://www.optimal-online.de:80/product_1.htm because the "Kopierschutzverfahren" link leads to the copy-x page, so please use that as the first known reference to copy-X. You can then add the known "Original_Website" as "First known description of copy-X". Finally, please also add http://web.archive.org/web/20110126034444/http://www.optimal-online.de/fileadmin/user_upload/PDF/copy_protection_july_09.pdf as the "last known official reference to copy-x" for 2011. Although the document was released in 2009, it being available until 2011 is significant for being the last known surviving reference to it. Also, please add the full dates of the archives as well.

Reply:
Should be fixed.